src/activity_actor.cpp

```diff
@@ -4856,7 +4864,7 @@ time_duration prying_activity_actor::prying_time( const activity_data_common &da
     int difficulty = pdata.difficulty;
     difficulty -= tool->get_quality( qual_PRY ) - pdata.prying_level;
     return time_duration::from_seconds(
-               /** @ARM_STR speeds up crowbar prying attempts */
+               /** @EFFECT_STR speeds up crowbar prying attempts */
```

Comment:
I would appreciate input on whether arm_str and lift_str and such should get unique aliases. They would probably still go on the main str page, like str_max does, but with different text.